lib/ansible/modules/database/mysql/mysql_replication.py

```diff
@@ -330,7 +356,9 @@ def start_slave(cursor, connection_name='', channel=''):
         executed_queries.append(query)
         cursor.execute(query)
         started = True
-    except Exception:
+    except Exception as e:
+        if fail_on_error:
+            module.fail_json(msg="START SLAVE failed: %s" % str(e))
```

Comment:
```suggestion
            module.fail_json(msg="START SLAVE failed: %s" % to_native(e))
```
please fix this in the other cases too

Reply:
Fixed - besides that, I fixed IP in integration test and improved comments of the tests there.